script.js

```diff
@@ -143,3 +143,16 @@ buttons.forEach((button, index) => {
     displayProject(projects[index]);
   });
 });
+
+const form = document.getElementById('form-portfolio');
+const email = form.elements.user_email;
+
+form.addEventListener('submit', () => {
+  const emailAddress = email.value;
+
+  if (emailAddress !== emailAddress.toLowerCase()) {
+    alert(
+      'The message was not sent, please put the email address in lower case. Thank you.',
+    );
```

Comment:
- Great Job for validating the email, however, you responded to the error message in the wrong way because of two reasons `alert` is not [javascript_best_practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) and it can be submitted after alert confirmation :+1: